test/nwise.js

```diff
@@ -0,0 +1,22 @@
+var assert = require('assert');
+
+var R = require('..');
+
+
+describe('nwise', function() {
+    var sevenLs = [1, 2, 3, 4, 5, 6, 7];
+    it('creates a list of n-tuples from a list', function() {
+        assert.deepEqual(R.nwise(3, sevenLs), [[1, 2, 3], [2, 3, 4], [3, 4, 5], [4, 5, 6], [5, 6, 7]]);
+        assert.deepEqual(R.nwise(4, [1, 2, 3, 4]), [[1, 2, 3, 4]]);
+    });
+
+    it('returns an empty list when `n` > `list.length`', function() {
+        assert.deepEqual(R.nwise(6, [1, 2, 3]), []);
```

Comment:
I'm not a big fan of omitting the ones outside of the chunk size. Its easier for the user to filter by length (i.e. remove the ones that don't fit the chunk size perfectly) than reconstruct the last set